### DISCUSSION TITLE
Setting up LC12S

### DISCUSSION THREAD

**Discussion by aachtenberg** [Yogui79/IntexPureSpa#3] [General]
Yogui,

Finally got my LC12S chips - they must have got quarantined for covid! I tried using your sketch in tools for finding the channel and it didn't seem to work. I used your initial sketch and was able to find the channel.
I am now searching for the network id. I will let you know how that goes.

**Comment by Yogui79** [comment]
Hello,

Witch sketch you use and doesn't work?

Witch one you use and work?

You need to find the channel before the network id and to put in inside the sketch to find the network id

**Comment by Yogui79** [comment]
Hi @aachtenberg 
I don't know what appends with your last commentar (I have it in a mail) but with the SearchChannel sketch you need to observe how the LC12S LED blink see this  [video](https://github.com/Yogui79/IntexPureSpa/blob/main/Docs/Video/ChannelSearchBlink.mp4)

**Reply by aachtenberg** [reply]
Yes, I deleted the comment because I did get it working. I switched to another LC12s I have (I ordered 5 just in case!). I do have everything working now - MQTT and can control the tub over a web browser.

**Reply by aachtenberg** [reply]
My channel is 0x14. Do the channels vary alot between spas?

**Reply by Yogui79** [reply]
Yes all spa have an other Channel until yet whatis your network id?

**Reply by aachtenberg** [reply]
#define USED_NETWORK_ID    0x266B
#define USED_CHANNEL       0x14
